pkg/bidstrategy/semantic/networking.go

```diff
@@ -17,13 +16,19 @@ func NewNetworkingStrategy(accept bool) *NetworkingStrategy {
 	return &NetworkingStrategy{accept}
 }
 
+const (
+	docsLink        = "(see https://docs.bacalhau.org/next-steps/networking)"
```

Comment:
I like this, we should definitely keep it, would be nice if we could do it with more errors too.

I think long-term using a hard-coded link might one day cause problems (either because it'll be wrong or we'll be managing redirects forever), so it might be worth a ticket to come up with a neat way of referencing the docs from the code.

Reply:
FWIW I think the discipline of maintaining redirects is no bad thing and is good for a healthy Web for other reasons. We should make it easy on ourselves though.